types/reveal/index.d.ts

```diff
@@ -2,7 +2,8 @@
 // Project: https://github.com/hakimel/reveal.js/
 // Definitions by: robertop87 <https://github.com/robertop87>,
 //                 Nava2 <https://github.com/Nava2>,
-//                 JPtenBerge <https://github.com/JPtenBerge>
+//                 JPtenBerge <https://github.com/JPtenBerge>,
+//                 farbodsz <https://github.com/farbodsz>
```

Comment:
Be aware of what this means - this is not just attribution. But happy to approve if you do want to be a definition owner.

https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
Thanks - yes I didn't mean to be added as a definition owner, I should've read that beforehand! Fixed now